metabase/metabase
# custom homepage modal

Closes https://github.com/metabase/metabase/issues/31097

### Description
Adds a toast for users who are redirected from the homepage to the dashboard that has been set. Once dismissed, this toast will not show again. This toast does not show for the user that made the change

### How to verify
1. As an admin, set up a dashboard to use as a custom homepage
2. log in as another user, you should see a toast explaining that you've been redirected
3. click "got it", and the toast should not appear the next time you are redirected

### Demo
![chrome_ivIFzrFzkB](https://github.com/metabase/metabase/assets/1328979/5e1e2f68-cd48-4885-a8e3-8a28037e22c3)


### Checklist
